### ISSUE TITLE
[Suggestion] A third option next to sentry and manual, where a unit will only shoot when a friendly unit is in danger (can be damaged)

### ISSUE THREAD

**Issue by Malakia1** [KatekovAnton/MAXTouch-docs#24]

**Comment by Malakia1** [comment]
Next to any unit having the possibility to be on sentry or not on sentry (manual) it would be great if there is a third option. 
In this third option a unit will not shoot at a unit that enters his attack range if that unit can not shoot at the defending unit or any other friendly unit. 

Currently it's a much used tactic to send cheap / harmless units into the enemies defense first, to then attack with units that can do damage.
(eg : air transports flown into anti air first, cheap units like scouts taking the hit against missile launchers before the big boys come in etc)

If we have this third option, it will greatly improve the possibilities of defending one's complex during the enemies turn.

**Comment by KatekovAnton** [comment]
🤔 @Malakia1 should they shoot on AWACs? or such units should not react on awac/apc/at/mt?

**Comment by Malakia1** [comment]
> 🤔 @Malakia1 should they shoot on AWACs? or such units should not react on awac/apc/at/mt?
No, when a unit is in this third mode, it should not shoot at AWACS or any other unit that can't shoot and only at units that can shoot when they actually can shoot at a friendly unit from their current position.
